### PR TITLE
Adds hat-wearing monkeys

### PR DESCRIPTION
**Pull request by Zymem**
## About the PR 
Adds the ability for monkeys to wear hats
**Screenshots**

image
image
^How the UI would look
**Changelog**

:cl: Zyme
- add: Monkeys now understand the concept of hats

### CODE REVIEW

File: Resources/Prototypes/InventoryTemplates/monkey_inventory_template.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- type: inventoryTemplate
+  id: monkey
+  slots:
+  - name: head
+    slotTexture: head
+    slotFlags: HEAD
+    uiContainer: BottomLeft
+    uiWindowPos: 0,0
+    displayName: Head
```
[the remark this answers is not in the document]
Comment (Zymem) [reply Resources/Prototypes/InventoryTemplates/monkey_inventory_template.yml:9]: Makes sense, I'll push the changes in a bit. I merged both templates into one called "head", not very original but it's the only name that I could come up with that made sense.

**Comment by mirrorcult** [comment]
conflict

**Review by mirrorcult — APPROVED** [review]

**Comment by Zymem** [comment]
I messed up. I'll close this one and open a new PR with an updated master.

**Comment by Zymem** [comment]
I just realized why the PR was conflicting.
Now that monkeys are no longer a ghost role, I don't think there's much point in giving them hats. If anyone wants to add them in the future, feel free to do it. The second commit should have the code necessary to add them.

**Comment by moonheart08** [comment]
> I just realized why the PR was conflicting. Now that monkeys are no longer a ghost role, I don't think there's much point in giving them hats. If anyone wants to add them in the future, feel free to do it. The second commit should have the code necessary to add them.

Monkeys are still ghost roles, they just are not ALWAYS a ghost role due to the random sentience event.

**Comment by Zymem** [comment]
> > I just realized why the PR was conflicting. Now that monkeys are no longer a ghost role, I don't think there's much point in giving them hats. If anyone wants to add them in the future, feel free to do it. The second commit should have the code necessary to add them.
> 
> Monkeys are still ghost roles, they just are not ALWAYS a ghost role due to the random sentience event.

Ohhh, that makes a lot of sense. Scratch that then, I'll make the new PR.

**Pull request closed without merging**